eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -101,6 +101,17 @@ steps:
 
   - ${{ parameters.BeforePublishSteps }}
 
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 'Upload Package SBOM'
+    inputs:
+      BuildDropPath: $(Build.ArtifactStagingDirectory)
+
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
```

Comment:
Do we need a sbom for the docs?

Reply:
I think so, yes. I figure it's a shipped artifact so 🤷‍♂️